statsmodels/stats/proportion.py

```diff
@@ -856,7 +856,7 @@ def proportions_ztest(count, nobs, value=None, alternative='two-sided',
     nobs_fact = np.sum(1. / nobs)
     if prop_var:
         p_pooled = prop_var
-    var_ = p_pooled * (1 - p_pooled) * nobs_fact
+    var_ = abs(p_pooled * (1 - p_pooled) * nobs_fact)
```

Comment:
It could still be 0. But I think it should raise if var_ is < 0, and possible <= 0.